plugins/woocommerce/src/Internal/Admin/ProductForm/Form.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * WooCommerce Product Form
+ *
+ * @package Woocommerce ProductForm
+ */
+
+namespace Automattic\WooCommerce\Internal\Admin\ProductForm;
+
+/**
+ * Contains logic for the WooCommerce Product Form.
+ */
+class Form {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var Form instance
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Store form fields.
+	 *
+	 * @var array
+	 */
+	protected static $form_fields = array();
+
+	/**
+	 * Get class instance.
+	 */
+	final public static function instance() {
+		if ( ! static::$instance ) {
+			static::$instance = new static();
+		}
+		return static::$instance;
+	}
+
+	/**
+	 * Init.
+	 */
+	public function init() {    }
+
+	/**
+	 * Adds a child menu item to the navigation.
+	 *
+	 * @param string $id Field id.
+	 * @param string $plugin_id Plugin id.
+	 * @param array  $args Array containing the necessary arguments.
+	 *     $args = array(
```

Comment:
According to https://github.com/woocommerce/woocommerce/issues/36019 we should also include a `label` property, right?